src/dataExplorer/components/NewDataExplorer.tsx

```diff
@@ -19,20 +20,22 @@ const NewDataExplorer: FC = () => {
 
   return (
     <QueryProvider>
-      <DraggableResizer
-        handleOrientation={Orientation.Vertical}
-        handlePositions={vertDragPosition}
-        onChangePositions={setVertDragPosition}
-      >
-        <DraggableResizer.Panel>
-          <Schema />
-        </DraggableResizer.Panel>
-        <DraggableResizer.Panel className="new-data-explorer-rightside">
-          <ResultsProvider>
-            <ResultsPane />
-          </ResultsProvider>
-        </DraggableResizer.Panel>
-      </DraggableResizer>
+      <EditorProvider>
```

Comment:
Only added the EditorProvider.
Nothing else changed.

Reply:
Because all injection into the editor will come through the `EditorContext.injectViaLsp()`.
This includes schema injection.
This is the lowest point in the hierarchy which also includes the schema browser.